gramhopper/configuration.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+from sys import platform
+import os
+
+
+_LINUX_GLOBAL_CONFIGURATION_DIR = '/etc/gramhopper'
+_LINUX_USER_CONFIGURATION_DIR = Path(Path.home(), '.gramhopper/')
+
+
+def configuration_dir():
+    if platform == "linux" or platform == "linux2":
+        if os.access(_LINUX_GLOBAL_CONFIGURATION_DIR, os.R_OK):
```

Comment:
Shouldn't we put the conf dir in ~/.gramhopper instead of /etc/gramhopper even if the user has access to the global dir?
It will be easier to write user guides, and nothing will break if the user will lose his access.
What do you think?

Reply:
Hmm... I thought this is better but now I'm not sure. Maybe we should consult a third person. Also, we should probably talk about that face to face so I'll be able to explain better.